src/borrower/deployer.sol

```diff
@@ -167,6 +168,9 @@ contract BorrowerDeployer {
 
             NFTFeedLike(nftFeed).init();
             DependLike(shelf).depend("subscriber", address(nftFeed));
+        } else {
+            // pool needs pile
+             DependLike(pricePool).depend("pile", address(pile));
```

Comment:
In the if and in the else is the exact same command:

`DependLike(nftFeed).depend("pile", address(pile));` is the same as ` DependLike(pricePool).depend("pile", address(pile));`